### PR TITLE
14 - API Endpoint: Add order

### PR DESCRIPTION
**Pull request by Baavi**

### CODE REVIEW

File: routes/addOrders.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import express from "express";
+import { db } from "../config/db.js";
+
+const router = express.Router();
+router.post("/addorder", async (_, res) => {
```
Comment (ShiroKatake) [inline routes/addOrders.js]: parameter needs to be `(req, res)` not `(_,res)`

File: routes/addOrders.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  try {
+    let transaction_id = req.body.transaction_id;
+    let member_id = req.body.member_id;
+    const sqlQuery =
+      "INSERT INTO Transaction (transaction_id, member_id) VALUES ('transaction_id', 'member_id');";
```
Comment (ShiroKatake) [inline routes/addOrders.js]: look at https://github.com/GoToGroMSP-20/GoToGroMSP-20-backend/pull/4 to know what the columns are. You're missing a few columns

**Comment by Baavi** [comment]
Resolve issues

File: routes/addOrders.js (hunk 2 above)
Comment (ShiroKatake) [reply routes/addOrders.js]: you're still missing a column to insert here

**Review by ShiroKatake — CHANGES_REQUESTED** [review]
Still need 1 last thing before this is good to merge

**Review by ShiroKatake — APPROVED** [review]
Naice job 👍

**Pull request merged**